crossvalidation.py

```diff
@@ -2,123 +2,188 @@
 import numpy as np
 import pandas as pd
 import json
+import argparse
+from hyperopt import hp, fmin, tpe, space_eval
```

Comment:
I like to do `hyperopt.hp` etcetera,  so it's clear where the functions are coming from; is there a reason to do otherwise here?